extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -248,43 +267,69 @@ export class AmpAutocomplete extends AMP.BaseElement {
    * @private
    */
   selectHandler_(event) {
-    if (!this.isItemElement_(event.target)) {
-      return Promise.resolve();
-    }
-    return this.mutateElement(() => {
-      this.selectItem_(event.target);
+    let item;
+    return this.measureMutateElement(() => {
+      item = this.getItemElement_(event.target);
```

Comment:
It looks like `getItemElement_` only reads from `classList`, which does not need to be wrapped in a `measure` block

Reply:
There's a branch where `getItemElement_` might read `element.parentElement`, won't that force reflow?

Reply:
Nope, `parentElement` won't force reflow. Reflow is generally caused by layout-related properties like widths, heights, offsets, etc and `parentNode` is information that the browser knows about through the DOM tree without needing to make sure that other elements layouts haven't changed. I had to look it up to verify though : P I usually use these two sources:

1. https://gist.github.com/paulirish/5d52fb081b3570c81e3a
2. https://github.com/ampproject/amphtml/blob/99d8eac351a94119233295dfda497b0b31b3967f/build-system/tasks/presubmit-checks.js#L682-L726